web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -387,7 +387,40 @@ class SpeakPage extends React.Component<Props, State> {
         let retries = 3;
         while (retries) {
           try {
-            await api.uploadClip(recording.blob, sentence.id, sentence.text);
+            const res = await api.uploadClip(
+              recording.blob,
+              sentence.id,
+              sentence.text
+            );
+            if (res.achievement) {
+              addNotification(
+                <div className="achievement">
+                  <img
+                    src={require('../../dashboard/challenge/images/star.svg')}
+                    alt=""
+                  />
+                  <p className="score">+ 50 points</p>
+                  <p>
+                    You're on your way! Congrats on your first contribution.{' '}
+                  </p>
+                </div>
+              );
```

Comment:
The same to the above

Reply:
To clarify that: show the first contribution toast if the piggy-backed `firstContribute` is true.
And `firstContribute` can only be true if exact one the following conditions is true:
1) uploaded clip is the first clips that the user has uploaded since the challenge's `start_date`;
2)the vote is the first vote that the user has voted since the challenge's `start_date`.